src/models2/ContinueModel.es6.js

```diff
@@ -0,0 +1,30 @@
+import RedditModel from './RedditModel';
+import Record from '../apiBase/Record';
+
+import { COMMENT_CONTINUE_THREAD } from './thingTypes';
+
+const T = RedditModel.Types;
+
+export default class ContinueThreadModel extends RedditModel {
+  static type = COMMENT_CONTINUE_THREAD;
+
+  static PROPERTIES = {
+    parentId: T.string,
+    id: T.string,
+    depth: T.number,
+  }
+
+  static API_ALIASES = {
+    parent_id: 'parentId',
+  }
+
+  makeUUID() {
+    return (Math.random() * 16).toFixed();
```

Comment:
Could this be `${this.parentId}-${COMMENT_CONTINUE_THREAD}`? Would be nice to make this deterministic, and avoid the (low, but still possible) chance of collision.